consensus/consensus_viewchange_msg.go

```diff
@@ -29,8 +29,10 @@ func (consensus *Consensus) constructViewChangeMessage() []byte {
 	// next leader key already updated
 	vcMsg.LeaderPubkey = consensus.LeaderPubKey.Serialize()
 
-	preparedMsgs := consensus.PbftLog.GetMessagesByTypeSeqHash(msg_pb.MessageType_PREPARED, consensus.blockNum, consensus.blockHash)
-	preparedMsg := consensus.PbftLog.FindMessageByMaxViewID(preparedMsgs)
+	preparedMsgs := consensus.PBFTLog.GetMessagesByTypeSeqHash(
+		msg_pb.MessageType_PREPARED, consensus.blockNum, consensus.blockHash,
```

Comment:
Is this kind of format change automatically by editor plugin?

Reply:
I changed it by hand because was wider than 100 col, code running off side of page.